### ISSUE TITLE
Server uptime never at "Starting"

### ISSUE THREAD

**Issue by AndyIsHereBoi** [pterodactyl/panel#4209]
### Current Behavior

The uptime of a server when in the console tab never will switch to "Starting". It always is offline or the uptime.
![image](https://user-images.githubusercontent.com/69745145/176333765-034f3357-d552-4f92-ad88-fc643977666f.png)
in the above image, the server uptime should show as starting as it in in the process of booting up.

### Expected Behavior

The text should change to "Starting" when the server is starting up.

### Steps to Reproduce

Start any server

### Panel Version

1.9.1

### Wings Version

1.6.4

### Is there an existing issue for this?

- [X] I have searched the existing issues before opening this issue.
- [X] I have provided all relevant details, including the specific game and Docker images I am using if this issue is related to running a server.
- [X] I have checked in the Discord server and believe this is a bug with the software, and not a configuration issue with my specific system.

**Comment by parkervcp** [comment]
If it is stuck at "starting" then you have a mis-configured startup phrase. All your screenshot shows is that it's showing offline before the container is started.

**Comment by Boy132** [comment]
> The uptime of a server when in the console tab never will switch to "Starting". It always is offline or the uptime.

I believe this is intentional - or at least it's no bug. After all it's the Uptime and not the Status block. So you're making a feature request, aren't you?

**Comment by itrack** [comment]
I encounter the same problem, serverul apare offline

**Comment by AndyIsHereBoi** [comment]
> > The uptime of a server when in the console tab never will switch to "Starting". It always is offline or the uptime.
> 
> I believe this is intentional - or at least it's no bug. After all it's the Uptime and not the Status block. So you're making a feature request, aren't you?

I was not sure if it was suppose to be like that or if it was a bug. If you want it can be a feature request because I still don't know if it was intentional. 
It also forced me to close the issue when i made the comment

**Comment by TekExplorer** [comment]
> > > The uptime of a server when in the console tab never will switch to "Starting". It always is offline or the uptime.
> > 
> > 
> > I believe this is intentional - or at least it's no bug. After all it's the Uptime and not the Status block. So you're making a feature request, aren't you?
> 
> I was not sure if it was suppose to be like that or if it was a bug. If you want it can be a feature request because I still don't know if it was intentional. It also forced me to close the issue when i made the comment

theres 2 buttons. comment, and close with comment. you pressed close with comment

**Comment by AndyIsHereBoi** [comment]
re open

**Comment by itrack** [comment]
<img width="1310" alt="Screenshot 2022-06-30 at 12 48 56" src="https://user-images.githubusercontent.com/2368391/176647776-30c58aa5-b2d8-492c-86fc-f3ad2ec5794b.png">
in my case the console is not displayed either, but the server is working, I can connect to it

**Comment by Boy132** [comment]
> <img alt="Screenshot 2022-06-30 at 12 48 56" width="1310" src="https://user-images.githubusercontent.com/2368391/176647776-30c58aa5-b2d8-492c-86fc-f3ad2ec5794b.png">
> 
> in my case the console is not displayed either, but the server is working, I can connect to it

That's not related this issue.
You should read the [troubleshooting guide](https://pterodactyl.io/panel/1.0/troubleshooting.html) and/ or ask on the discord.

**Comment by FC-FailCoder** [comment]
Should the uptime start counting:
1. when the server is started.
2. when the server is starting.

Just asking as at the moment I added both: Starting and Stopping into the uptime conditions - but also wanted to know what is considered uptime, when the server is in its starting state or when the server is ready to be used (running)?

**Comment by AndyIsHereBoi** [comment]
The running is when it's finished the pterodactyl stuff. Anytime before that after start is pressed I would consider it in a starting state

**Comment by trenutoo** [comment]
> Should the uptime start counting:
> 
>     1. when the server is started.
> 
>     2. when the server is starting.
> 
> 
> Just asking as at the moment I added both: Starting and Stopping into the uptime conditions - but also wanted to know what is considered uptime, when the server is in its starting state or when the server is ready to be used (running)?

All this requires is [expanding the ternary operation to not default to offline if uptime is 0](https://github.com/pterodactyl/panel/blob/develop/resources/scripts/components/server/console/ServerDetailsBlock.tsx#L100), which Wings will return if a server is not running. The panel displays the uptime retrieved from the Wings, The only thing Panel does to uptime is convert it from milliseconds to seconds.

e.g 

```
                {stats.uptime > 0 ? (
                    <UptimeDuration uptime={stats.uptime / 1000} />
                ) : (
                    <> {status === 'starting' ? 'Starting' : 'Offline'}</>
                )}
```

ED: The logic has to be inverted if we do take into account the stopping state, which I forgot exists.

**Comment by FC-FailCoder** [comment]
> > Should the uptime start counting:
> 
> > 
> 
> >     1. when the server is started.
> 
> > 
> 
> >     2. when the server is starting.
> 
> > 
> 
> > 
> 
> > Just asking as at the moment I added both: Starting and Stopping into the uptime conditions - but also wanted to know what is considered uptime, when the server is in its starting state or when the server is ready to be used (running)?
> 
> 
> 
> All this requires is [expanding the ternary operation to not default to offline if uptime is 0](https://github.com/pterodactyl/panel/blob/develop/resources/scripts/components/server/console/ServerDetailsBlock.tsx#L100), which Wings will return if a server is not running. The panel displays the uptime retrieved from the Wings, The only thing Panel does to uptime is convert it from milliseconds to seconds.
> 
> 
> 
> e.g 
> 
> 
> 
> ```
> 
>                 {stats.uptime > 0 ? (
> 
>                     <UptimeDuration uptime={stats.uptime / 1000} />
> 
>                 ) : (
> 
>                     <> {status === 'starting' ? 'Starting' : 'Offline'}</>
> 
>                 )}
> 
> ```
> 
> 
> 
> ED: The logic has to be inverted if we do take into account the stopping state, which I forgot exists.

Sorry I should've worded myself a bit better. With the logic I tried, which was if the server uptime is more than zero and is in the "running" state, then it would display the uptime (+ the logic you stated but with the stopping status).

This would cause it to show e.g 20 seconds already after the "Starting" was displayed ("starting" state). My question was in regards to: shouldn't the uptime start counting on the wings side when the server is considered "running".

I did this logic because that's what I thought the feature requested. I didn't think they wanted the feature request to be "Starting" for a second and then it changes to uptime (when still starting).

^ if this still doesn't make sense I'll reword it tomorrow as I am typing on my phone

**Comment by AndyIsHereBoi** [comment]
I think it should switch to starting once the user presses the start button, then when the uptime is greater then 1ms then it will switch to the running state
